test/fuzzTesting/fuzzHelper.cpp

```diff
@@ -143,16 +151,136 @@ std::string RandomCode::fillArguments(dev::eth::Instruction _opcode, RandomCodeO
+		case dev::eth::Instruction::PUSH32:	code += rndByteSequence(32); break;
+		case dev::eth::Instruction::SWAP1:
+		case dev::eth::Instruction::SWAP2:
+		case dev::eth::Instruction::SWAP3:
+		case dev::eth::Instruction::SWAP4:
+		case dev::eth::Instruction::SWAP5:
+		case dev::eth::Instruction::SWAP6:
+		case dev::eth::Instruction::SWAP7:
+		case dev::eth::Instruction::SWAP8:
+		case dev::eth::Instruction::SWAP9:
+		case dev::eth::Instruction::SWAP10:
+		case dev::eth::Instruction::SWAP11:
+		case dev::eth::Instruction::SWAP12:
+		case dev::eth::Instruction::SWAP13:
+		case dev::eth::Instruction::SWAP14:
+		case dev::eth::Instruction::SWAP15:
+		case dev::eth::Instruction::SWAP16:
+		case dev::eth::Instruction::DUP1:
+		case dev::eth::Instruction::DUP2:
+		case dev::eth::Instruction::DUP3:
+		case dev::eth::Instruction::DUP4:
+		case dev::eth::Instruction::DUP5:
+		case dev::eth::Instruction::DUP6:
+		case dev::eth::Instruction::DUP7:
+		case dev::eth::Instruction::DUP8:
+		case dev::eth::Instruction::DUP9:
+		case dev::eth::Instruction::DUP10:
+		case dev::eth::Instruction::DUP11:
+		case dev::eth::Instruction::DUP12:
+		case dev::eth::Instruction::DUP13:
+		case dev::eth::Instruction::DUP14:
+		case dev::eth::Instruction::DUP15:
+		case dev::eth::Instruction::DUP16:
+			int times;
+			switch (_opcode)
+			{
+				case dev::eth::Instruction::DUP1:	times = 1; break;
+				case dev::eth::Instruction::SWAP1:
+				case dev::eth::Instruction::DUP2:	times = 2; break;
+				case dev::eth::Instruction::SWAP2:
+				case dev::eth::Instruction::DUP3:	times = 3; break;
+				case dev::eth::Instruction::SWAP3:
+				case dev::eth::Instruction::DUP4:	times = 4; break;
+				case dev::eth::Instruction::SWAP4:
+				case dev::eth::Instruction::DUP5:	times = 5; break;
+				case dev::eth::Instruction::SWAP5:
+				case dev::eth::Instruction::DUP6:	times = 6; break;
+				case dev::eth::Instruction::SWAP6:
+				case dev::eth::Instruction::DUP7:	times = 7; break;
+				case dev::eth::Instruction::SWAP7:
+				case dev::eth::Instruction::DUP8:	times = 8; break;
+				case dev::eth::Instruction::SWAP8:
+				case dev::eth::Instruction::DUP9:	times = 9; break;
+				case dev::eth::Instruction::SWAP9:
+				case dev::eth::Instruction::DUP10:	times = 10; break;
+				case dev::eth::Instruction::SWAP10:
+				case dev::eth::Instruction::DUP11:	times = 11; break;
+				case dev::eth::Instruction::SWAP11:
+				case dev::eth::Instruction::DUP12:	times = 12; break;
+				case dev::eth::Instruction::SWAP12:
+				case dev::eth::Instruction::DUP13:	times = 13; break;
+				case dev::eth::Instruction::SWAP13:
+				case dev::eth::Instruction::DUP14:	times = 14; break;
+				case dev::eth::Instruction::SWAP14:
+				case dev::eth::Instruction::DUP15:	times = 15; break;
+				case dev::eth::Instruction::SWAP15:
+				case dev::eth::Instruction::DUP16:	times = 16; break;
+				case dev::eth::Instruction::SWAP16:	times = 17; break;
+				default: times = 1;
+			}
+			for (int i = 0; i < times; i ++)
```

Comment:
it might happen. cause some previous random code might have puted smth to memory.
in this switch case I am filling the stack for opcode. it is containing addresses to memory.
the actual memory could be filled with smth or not.